go/vt/zktopo/zktestserver/zktopo_test.go

```diff
@@ -1,86 +0,0 @@
-package zktestserver
-
-import (
-	"path"
-	"testing"
-	"time"
-
-	zookeeper "github.com/samuel/go-zookeeper/zk"
-	"golang.org/x/net/context"
-
-	"github.com/youtube/vitess/go/vt/topo"
-	"github.com/youtube/vitess/go/vt/topo/test"
-	"github.com/youtube/vitess/go/vt/zktopo"
-	"github.com/youtube/vitess/go/zk"
-
-	topodatapb "github.com/youtube/vitess/go/vt/proto/topodata"
-)
-
-func TestZkTopo(t *testing.T) {
-	zktopo.WatchSleepDuration = 2 * time.Millisecond
-	test.TopoServerTestSuite(t, func() topo.Impl {
```

Comment:
This is going to remove the code coverage for the old "zookeeper" implementation, isn't it?